### ISSUE TITLE
Could I use the "rotate" in prod environment？

### ISSUE THREAD

**Issue by zhaidongbo** [onkel-dirtus/logger_file_backend#65]
I find the configure of "rotate" in code, but not in README.md, so could the "rotate" be used in prod environment？

**Comment by fireproofsocks** [comment]
Yes, rotation can be used in production. Log rotation is fully supported and documented in v0.1.1. See the README for configuration examples: https://hex.pm/packages/logger_file_backend/0.1.1
